libfreerdp/core/gateway/tsg.c

```diff
@@ -674,7 +661,6 @@ BOOL TsProxyAuthorizeTunnelReadResponse(rdpTsg* tsg, RPC_PDU* pdu)
 	
 	if (SizeValue == 4)
 	{
-		idleTimeout = *((UINT32*) &buffer[offset]);
```

Comment:
Again, even if we are not using this IdleTimeout value, it is there as a placeholder for now. I actually had to spend some time figuring out where that unparsed IdleTimeout value was being sent by the server, not having a placeholder will make the next developer waste time figuring out where it should be parsed from and how.